### ISSUE TITLE
Problems after updating to coveralls 0.8.13

### ISSUE THREAD

**Issue by kzarnow** [xing/hardcover#11]
After updating `gem coveralls` from version 0.7.3 to 0.8.13 we start to get following problems and nothing is sent to hardcover:
```
18:19:13 [36m[Coveralls] Submitting to https://MY_HARDCOVER_SERVER_HOSTNAME/api/v1[0m
18:19:13 [36m[Coveralls] can't be blank[0m
```
Could you help me with that, please?

**Comment by pietbrauer** [comment]
Did you try using our fork? https://github.com/xing/hardcover#rails

**Comment by kzarnow** [comment]
Yes, we are using that one already

**Comment by pietbrauer** [comment]
But then it is beyond me how you could upgrade to 0.8.3 because the latest version is 0.7.0 https://github.com/xing/coveralls-ruby/releases

**Comment by kzarnow** [comment]
Yes, you are totally right, I misunderstood your previous message, sorry! 
I used https://github.com/lemurheavy/coveralls-ruby/releases/tag/v0.8.13
I came to that issue because in coveralls dependency `rest-client` security issues were reported (https://github.com/rest-client/rest-client/issues/369), an I noticed that in `coveralls 0.8.11` this dependency was removed

**Comment by pietbrauer** [comment]
If you want to update the dependencies we would welcome a Pull Request to [xing/coveralls-ruby](https://github.com/xing/coveralls-ruby).
